setup/common/040_ValidateSignCert.rb

```diff
@@ -21,7 +21,7 @@
   hosts.each do |host|
     ssldir = on(host, puppet('agent --configprint ssldir')).stdout.chomp
     # preserve permissions for master's ssldir so puppetserver can read it
-    on(host, "rm -rf '#{ssldir}/*'")
+    on(host, "rm -rf #{ssldir}/*")
```

Comment:
I think we added quotes originally because some older Windows versions had spaces, e.g. "Application Data". Will this still work on supported Windows platforms?